en/docs/assets/js/theme.js

```diff
@@ -144,6 +143,48 @@ request.onload = function() {
                 .setAttribute('href', docSetUrl + 'versions');
         }
 
+             'versions/assets/versions.json', true);
+
+request.onload = function() {
```

Comment:
@chamindi-a let's remove this change in a new PR as this is a duplicate of an existing function in the same file. This breaks the version selection dropdown of the documentation after someone switches to 4.1.0.